proto/ibc/applications/interchain_accounts/controller/v1/tx.proto

```diff
@@ -26,3 +30,23 @@ message MsgRegisterAccount {
 message MsgRegisterAccountResponse {
   string channel_id = 1 [(gogoproto.moretags) = "yaml:\"channel_id\""];
 }
+
+// MsgSubmitTx defines the payload for MsgSubmitTx
+message MsgSubmitTx {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string owner         = 1;
```

Comment:
This can also be cast to sdk.Address

Reply:
is it better to cast it in the proto or check if its a valid addr in ValidateBasic?

Reply:
I think it is fine to leave as is, primarily because the owner is also the signer and all of our Msg types have the signer as an uncasted string (albeit I'm not sure why)

Reply:
I think it makes sense just to apply the casting change to all our messages at once. The SDK does some [different](https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/group/v1/tx.proto#L315) things nowadays anyways

Reply:
I looked into this some more. Looks like the SDK intentionally chose to go with a string in 2020 (during amino -> proto migration). Might have been some concern with the gRPC gateway, nevertheless there's talk about removing reliance on the global bech32 config and moving it to be a param of the auth keeper, so you'd need access to the auth keeper to get the acc address (they want to change get signers to return a string instead of an address)

Lets keep with what we have. It'll be easier to deal with everything uniformly